ethereum/statetransition/src/main/java/tech/pegasys/artemis/statetransition/BeaconState.java

```diff
@@ -424,14 +426,15 @@ private int get_fork_version(Fork fork, int slot) {
   @VisibleForTesting
   public void activate_validator(BeaconState state, int index, boolean is_genesis) {
```

Comment:
If this is working now, then it would be good to fix the test in BeaconStateTest.java

Reply:
Working on it rn

Reply:
Done